input/full.go

```diff
@@ -64,15 +81,13 @@ func CollectFull(ctx context.Context, server *state.Server, connection *sql.DB,
 		ps.StatementResetCounter = 0
 		err = postgres.ResetStatements(ctx, logger, connection, systemType)
 		if err != nil {
-			// This is a non-fatal error, so continue snapshot collection but do log it as an error
-			// (this gets automatically added to the snapshot's CollectorErrors information)
 			logger.PrintError("Error calling pg_stat_statements_reset() as requested: %s", err)
 			err = nil
 		} else {
 			_, _, ts.ResetStatementStats, err = postgres.GetStatements(ctx, server, logger, connection, globalCollectionOpts, ts.Version, false, systemType)
 			if err != nil {
-				err = fmt.Errorf("Error collecting pg_stat_statements: %s", err)
-				return
+				logger.PrintError("Error collecting pg_stat_statements: %s", err)
```

Comment:
I wonder if it would be nice to explicitly mention "after reset". This error is the same message as the before-reset one but I'd say that this has a bit different nature